src/components/Booklist.js

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import BookItem from './BookItem';
+import { getBooksToDisplayApi } from '../redux/books/books';
 
 export default function BookList() {
   const booklist = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
```

Comment:
[OPTIONAL] When I run your app I get compile errors because of  A few typos that you have with your imports.

![image](https://user-images.githubusercontent.com/38088818/185469818-224e02b3-7aa1-44f2-85b5-8e74b50d0233.png)

- Here when you import `Bookstore` and `Categories` you used a small letter for `./pages/...` yet your directory is capitalized therefore it should be `./Pages/...`

Another error is with your import of `BookList`.

![image](https://user-images.githubusercontent.com/38088818/185470525-45f8b964-bc1b-401b-987b-15dcc6acfdb5.png)

-  Here in your path `../components/BookList` you used `BookList` instead of `Booklist` which is the correct way you named your component.

Kindly fix these compile errors to make sure that your app runs as expected.

Reply:
Thanks for your great review. I will fix the typo issues.